code/modules/vending/youtool.dm

```diff
@@ -22,10 +22,10 @@
 	armor = list("melee" = 100, "bullet" = 100, "laser" = 100, "energy" = 100, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 70)
 	refill_canister = /obj/item/vending_refill/tool
 	resistance_flags = FIRE_PROOF
-	default_price = 50
+	default_price = 10
```

Comment:
20. Still lower than the average greyman paycheck.

Reply:
Tools are literally, _literally_ everywhere, more abundant than food, and can be printed from autolathes, i see no reason why a higher / same price than food is needed when food is a greater scarcity.

Reply:
You are tempting me to remove some toolboxes from maps to justify the prices.